hack/sync-upstream.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash -ex
+
+DS_FILES=(OWNERS hack/sync-upstream.sh)
+EXCLUDE_FILES=$(echo ${DS_FILES[@]} | tr ' ' '|')
+
+US=($(git remote -v | grep -E $"^[^${TAB} ]*${TAB}.*kubernetes-sigs/cluster-api-provider-kubevirt.git" | cut -f1 | uniq))
+UPSTREAM=${US[0]}
+if [[ -z ${UPSTREAM} ]]; then
+  git remote add upstream https://github.com/kubernetes-sigs/cluster-api-provider-kubevirt.git
+  UPSTREAM=upstream
+fi
+
+OS=($(git remote -v | grep -E $"^[^${TAB} ]*${TAB}.*openshift/cluster-api-provider-kubevirt.git" | cut -f1 | uniq))
```

Comment:
Why do you need to check if `openshift/cluster-api-provider-kubevirt.git` exists
Doesn't it must be exists as it run as part of the openshift org repository?

Reply:
This line checks that local repo is connected to the downstream repo - that there is a remote defined for the downstream repo.